back/engines/commercial/admin_api/spec/acceptance/projects_spec.rb

```diff
@@ -57,19 +59,25 @@
       end
     end
 
-    example 'Import a project template' do
-      do_request(tenant_id: tenant.id, project: { template_yaml: template.to_yaml, folder_id: folder.id })
+    describe 'Import a project template' do
+      example 'it imports a project' do
+        do_request(tenant_id: tenant.id, project: { template_yaml: template.to_yaml, folder_id: folder.id })
+        expect(status).to eq(200)
 
-      expect(status).to eq(200)
-      expect(DumpTenantJob).to have_been_enqueued if defined?(NLP)
+        tenant.switch do
+          project = Project.first
 
-      tenant.switch do
-        project = Project.first
+          expect(template['models']['project'].first.dig('title_multiloc', 'en')).to eq project.title_multiloc['en']
+          expect(template['models']['phase'].size).to eq project.phases.count
+          expect(template['models']['phase'].pluck('start_at')).to match_array project.phases.map(&:start_at).map(&:iso8601)
+          expect(project.folder_id).to eq folder.id
+        end
+      end
 
-        expect(template['models']['project'].first.dig('title_multiloc', 'en')).to eq project.title_multiloc['en']
-        expect(template['models']['phase'].size).to eq project.phases.count
-        expect(template['models']['phase'].pluck('start_at')).to match_array project.phases.map(&:start_at).map(&:iso8601)
-        expect(project.folder_id).to eq folder.id
+      if defined?(NLP)
+        example 'it enqueues DumpTenantJob once if NLP defined' do
+          expect { do_request(tenant_id: tenant.id, project: { template_yaml: template.to_yaml, folder_id: folder.id }) }.to have_enqueued_job(DumpTenantJob)
+        end
```

Comment:
This change is an attempt to avoid the flakiness I was experiencing with this test, where it would sometimes fail because DumpTenantJob was enqueued more than once (probably because my other new tests also invoke enqueueing this job).

@kogre Suggested passing a block to `expect { ... }.to have_enqueued ...`, as this seems to be the approach used in the [rspec docs](https://relishapp.com/rspec/rspec-rails/docs/matchers/have-enqueued-job-matcher).

Reply:
It seems we indeed don't clear the jobs in tests. If you don't mind fixing it for the future, could you try running `ActiveJob::Base.queue_adapter.enqueued_jobs.clear` in `before(:each)` block in your test?

If it works, it makes sense to add it in `back/spec/spec_helper.rb:179`

Reply:
Sure. But let's do this in the next BE PR for this epic. I'd like to get this merged into the feature branch.